projects/deja-js/component/editor/deja-editor.component.ts

```diff
@@ -387,10 +387,13 @@ export class DejaEditorComponent extends Destroy implements OnChanges, AfterView
     }
 
     public setFocus(): void {
-        if (this.instance) {
+        if (this.instance?.status === 'ready') {
             this.instance.focus();
         } else {
-            this.host.nativeElement.focus();
+            this.ready.pipe(
```

Comment:
Comme expliqué à la séance déja, pas de subscribe dans une fonction
Créer un sujet setFocus, switchmapper sur of(this.instance) ou this.ready.pipe(take(1)), et faire le this.instance.focus() à la fin